src/DefaultRESTClient.ts

```diff
@@ -14,10 +14,10 @@
  * language governing permissions and limitations under the License.
  */
 
-import IRESTClient, { ErrorResponseHandler, ResponseHandler } from "./IRESTClient";
+import IRESTClient, {ErrorResponseHandler, ResponseHandler} from "./IRESTClient";
```

Comment:
We need to get everyone synchronized on the typescript style because these lines should not change